datasets/indonlu/indonlu.py

```diff
@@ -0,0 +1,644 @@
+                    if self.config.name == "emot":
+                        label, tweet = row
+                        yield id_, {"tweet": tweet, "label": label}
+                    elif self.config.name == "wrete":
+                        premise, hypothesis, category, label = row
+                        yield id_, {"premise": premise, "hypothesis": hypothesis, "category": category, "label": label}
+                    elif self.config.name == "facqa":
+                        question, passage, seq_label = row
+                        yield id_, {
+                            "question": ast.literal_eval(question),
+                            "passage": ast.literal_eval(passage),
+                            "seq_label": ast.literal_eval(seq_label),
+                        }
+                    elif self.config.name == "casa" or self.config.name == "hoasa":
+                        sentence, *labels = row
+                        sentence = {"sentence": sentence}
+                        label = {l: labels[idx] for idx, l in enumerate(self.config.label_column)}
+                        yield id_, {**sentence, **label}
+            elif self.config.name in tsv_file:
+                reader = csv.reader(f, delimiter="\t", quoting=csv.QUOTE_NONE)
+
+                for id_, row in enumerate(reader):
+                    if self.config.name == "smsa":
+                        text, label = row
+                        yield id_, {"text": text, "label": label}
+            elif self.config.name in (txt_file + txt_file_pos + txt_file_ner):
+                id_ = 0
+                tokens = []
+                seq_label = []
+                for line in f:
+                    if len(line.strip()) > 0:
+                        token, label = line[:-1].split("\t")
+                        tokens.append(token)
+                        seq_label.append(label)
+                    else:
+                        if self.config.name in txt_file:
+                            yield id_, {"tokens": tokens, "seq_label": seq_label}
+                        elif self.config.name in txt_file_pos:
+                            yield id_, {"tokens": tokens, "pos_tags": seq_label}
+                        elif self.config.name in txt_file_ner:
+                            yield id_, {"tokens": tokens, "ner_tags": seq_label}
+                        id_ += 1
+                        tokens = []
+                        seq_label = []
+                    # add last example
+                    if tokens:
+                        if self.config.name in txt_file:
+                            yield id_, {"tokens": tokens, "seq_label": seq_label}
+                        elif self.config.name in txt_file_pos:
+                            yield id_, {"tokens": tokens, "pos_tags": seq_label}
+                        elif self.config.name in txt_file_ner:
+                            yield id_, {"tokens": tokens, "ner_tags": seq_label}
```

Comment:
```suggestion
                # add last example
                if tokens:
                    if self.config.name in txt_file:
                        yield id_, {"tokens": tokens, "seq_label": seq_label}
                    elif self.config.name in txt_file_pos:
                        yield id_, {"tokens": tokens, "pos_tags": seq_label}
                    elif self.config.name in txt_file_ner:
                        yield id_, {"tokens": tokens, "ner_tags": seq_label}
```
this has to be outside the for loop

Reply:
Alright, I've made the last 2 changes @lhoestq.